### PR TITLE
Python 3 syntax fixes

### PR DESCRIPTION
**Pull request by travislbrundage**
A few more Python 3 syntax fixes and removal of remaining Python 2 compatibility. Follow up to https://github.com/GeoNode/geonode/pull/5740/ and including some of https://github.com/GeoNode/geonode/pull/5741 which was not captured.

## Checklist

> Reviewing is a process done by project maintainers, mostly on a volunteer basis. We try to keep the overhead as small as possible and appreciate if you help us to do so by completing the following items. Feel free to ask in a comment if you have troubles with any of them.

For all pull requests:

- [x] Confirm you have read the [contribution guidelines](https://github.com/GeoNode/geonode/blob/master/CONTRIBUTING.md) 
- [x] You have sent a Contribution Licence Agreement (CLA) as necessary (not required for small changes, e.g., fixing typos in documentation)
- [x] Make sure the first PR targets the master branch, eventual backports will be managed later. This can be ignored if the PR is fixing an issue that only happens in a specific branch, but not in newer ones.

The following are required only for core and extension modules (they are welcomed, but not required, for contrib modules):
- [x] There is a ticket in https://github.com/GeoNode/geonode/issues describing the issue/improvement/feature (a notable exemptions is, changes not visible to end users)
- [ ] The issue connected to the PR must have Labels and Milestone assigned
- [x] PR for bug fixes and small new features are presented as a single commit
- [ ] Commit message must be in the form "[Fixes #<issue_number>] Title of the Issue"
- [x] New unit tests have been added covering the changes, unless there are explanation on why the tests are not necessary/implemented
- [x] This PR passes all existing unit tests (test results will be reported by travis-ci after opening this PR)
- [x] This PR passes the QA checks: flake8 geonode
- [x] Commits changing the **settings**, **UI**, **existing user workflows**, or adding **new functionality**, need to include documentation updates

**Submitting the PR does not require you to check all items, but by the time it gets merged, they should be either satisfied or inapplicable.**

### CODE REVIEW

File: geonode/people/migrations/0026_auto_20171120_0730.py [hunk 1]
```diff
@@ -14,7 +14,7 @@ class Migration(migrations.Migration):
...
...0\u0940'), ('my', '\u1019\u103c\u1014\u103a\u1019\u102c\u1018\u102c\u101e\u102c'), ('nb', 'norsk (bokm\xe5l)'), ('ne', '\u0928\u0947\u092a\u093e\u0932\u0940'), ('nl', 'Nederlands'), ('nn', 'norsk (nynorsk)'), ('os', '\u0418\u0440\u043e\u043d'), ('pa', 'Punjabi'), ('pl', 'polski'), ('pt', 'Portugu\xeas'), ('pt-br', 'Portugu\xeas Brasileiro'), ('ro', 'Rom\xe2n\u0103'), ('ru', '\u0420\u0443\u0441\u0441\u043a\u0438\u0439'), ('sk', 'slovensk\xfd'), ('sl', 'Sloven\u0161\u010dina'), ('sq', 'shqip'), ('sr', '\u0441\u0440\u043f\u0441\u043a\u0438'), ('sr-latn', 'srpski (latinica)'), ('sv', 'svenska'), ('sw', 'Kiswahili'), ('ta', '\u0ba4\u0bae\u0bbf\u0bb4\u0bcd'), ('te', '\u0c24\u0c46\u0c32\u0c41\u0c17\u0c41'), ('th', '\u0e20\u0e32\u0e29\u0e32\u0e44\u0e17\u0e22'), ('tr', 'T\xfcrk\xe7e'), ('tt', '\u0422\u0430\u0442\u0430\u0440\u0447\u0430'), ('udm', '\u0423\u0434\u043c\u0443\u0440\u0442'), ('uk', '\u0423\u043a\u0440\u0430\u0457\u043d\u0441\u044c\u043a\u0430'), ('ur', '\u0627\u0631\u062f\u0648'), ('vi', 'Ti\xea\u0301ng Vi\xea\u0323t'), ('zh-cn', '\u7b80\u4f53\u4e2d\u6587'), ('zh-hans', '\u7b80\u4f53\u4e2d\u6587'), ('zh-hant', '\u7e41\u9ad4\u4e2d\u6587'), ('zh-tw', '\u7e41\u9ad4\u4e2d\u6587')]),
```
Comment (travislbrundage) [inline geonode/people/migrations/0026_auto_20171120_0730.py]: I'm unsure about these changes. They were made in the previous PR, and the `2to3` script does suggest it. However it looks to me like a single backslash escapes the UTF character properly, not double - is there some reason it's supposed to be double backslash?

Comment (afabiani) [reply geonode/people/migrations/0026_auto_20171120_0730.py]: Uhm, since it was auto-generated I would leave this as it was originally.

**Comment by afabiani** [comment]
It looks like the build is failing due to some flake8 issues

```
flake8 geonode
geonode/geoserver/tests/test_server.py:622:31: E127 continuation line over-indented for visual indent
geonode/geoserver/tests/test_server.py:866:28: E127 continuation line over-indented for visual indent
geonode/geoserver/tests/test_server.py:870:27: E127 continuation line over-indented for visual indent
```

**Comment by travislbrundage** [comment]
Fixed the flake8 errors.

File: geonode/people/migrations/0026_auto_20171120_0730.py (hunk 1 above)
Comment (travislbrundage) [reply geonode/people/migrations/0026_auto_20171120_0730.py]: Okay, that is what I thought too. So this change reverts it back. Thanks!

**Pull request merged**